packages/google_sign_in/README.md

```diff
@@ -21,11 +21,12 @@ enable the [Google People API](https://developers.google.com/people/).
 ## iOS integration
 
 1. [First register your application](https://developers.google.com/mobile/add?platform=ios).
-2. Open Xcode. You'll have to paste this into Xcode to properly register `GoogleServices-Info.plist`.
-3. Select `GoogleServices-Info.plist` from the file manager and drag that file into the `Runner` directory, `[my_project]/ios/Runner/GoogleServices-Info.plist`.
-4. A dialog will show up and ask you to select the targets, select the `Runner` target.
-5. Then add the `CFBundleURLTypes` attributes below into the `[my_project]/ios/Runner/Info.plist` file.
-
+2. Make sure the file you download in step 1 is named `GoogleService-Info.plist`, rename it if necessary.
+3. Move or copy `GoogleService-Info.plist` into the `[my_project]/ios/Runner` directory.
```

Comment:
The old instruction seems fair enough. I think we only need to add the `Make sure the file you download in step 1 is named ...` part.